CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 ## 2.8.0 (Unreleased)
 
+* Changed signature for `FindAdminCatalogRecords`, which now returns normalized type `[]*types.CatalogRecord`
+* Added methods `catalog.QueryVappTemplateList`, `catalog.QueryCatalogItemList`, `client.queryWithMetadataFields`, `client.queryByMetadataFilter`
+* Added query engine based on `client.SearchByFilter`, type `FilterDef`, and interface `QueryItem`
+* Added methods `adminOrg.QueryCatalogList` and `org.QueryCatalogList`
```

Comment:
Just noticed that URL links to the PR are missing. They are esp. important here given the size and complexity of the PR.